.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Please include .vscode as well as other config files not related to the actual project into .gitignore